app/_locales/en/messages.json

```diff
@@ -2648,6 +2648,12 @@
   "usePhishingDetectionDescription": {
     "message": "Display a warning for phishing domains targeting Ethereum users"
   },
+  "useStaticTokenList": {
+    "message": "Token detection"
+  },
+  "useStaticTokenListDescription": {
+    "message": "We use third-party APIs to detect and display new tokens sent to your wallet. Turn off if you don’t want the app to pull data from those those services."
```

Comment:
nit: Just a suggestion, but maybe we could adjust the wording to `We use third-party APIs to detect and display new tokens sent to your wallet. Turn this off if you don’t want MetaMask to pull data from those those services.`?